src/trpc-client.ts

```diff
@@ -0,0 +1,37 @@
+import { httpBatchLink } from "@trpc/client";
+import { createTRPCNext } from "@trpc/next";
+import { AppRouter } from "./server/routers/app-router";
+import { SALEOR_AUTHORIZATION_BEARER_HEADER, SALEOR_DOMAIN_HEADER } from "@saleor/app-sdk/const";
+import { appBridgeInstance } from "./pages/_app";
+
+
+function getBaseUrl() {
+  if (typeof window !== "undefined") return "";
```

Comment:
Shouln't we throw an error if this function is used outside of the browser context instead of returning an empty string?